var/spack/repos/builtin/packages/hypre/package.py

```diff
@@ -374,8 +374,10 @@ def install(self, spec, prefix):
             make("install")
             if spec.satisfies("+gptune"):
                 make("test")
-                self.run_test("mkdir", options=["-p", self.prefix.bin])
-                self.run_test("cp", options=["test/ij", self.prefix.bin + "/."])
+                mkdir = which("mkdir")
+                mkdir("-p", self.prefix.bin)
```

Comment:
You can use the spack-provided `mkdirp`.

```suggestion
                mkdirp(self.prefix.bin)
```